config/default/common/config/metadata/stories/black_marble_night_lights/step004.md

```diff
@@ -0,0 +1 @@
+A major power outage occurred in Houston, Texas on 16 February 2021 due to severe winter weather. On the left is an image from before the storm on 7 February 2021, the city lights fill up most of the central area and emanate outwards along major roads. Move the center swipe line to the left and dark patches appear in the western, eastern, and southern portions of the city on 16 February 2021 where power outages affected the city for several days. In this comparison, we've applied a different color palette to highlight the differences between the two images. You can change the color palette by clicking on the "View Options" icon on the layer and select from a range of different color palettes.
```

Comment:
--> "In this comparison, we've applied a different color palette to highlight the differences between the brighter and darker areas."